dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json

```diff
@@ -15,7 +15,7 @@
       },
       "base": {
         "type": "string",
-        "description": "Reference currency."
+        "description": "Reference currency. This value must match exactly (including capitalization) one of the currency abbreviations found <a href=\"https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html\">here</a>."
```

Comment:
what about:
ISO reference currency. See <a href=\"https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html\">here</a> for an exhaustive list.